fastly/block_fastly_service_v1_gzip.go

```diff
@@ -118,33 +106,19 @@ func (h *GzipServiceAttributeHandler) Process(d *schema.ResourceData, latestVers
 			Name:           resource["name"].(string),
 		}
 
+		// NOTE: []interface{} is not comparable in Filter function
+		// covert it into string in advance
```

Comment:
Just noticed this typo here as well
```suggestion
		// convert it into string in advance
```

Reply:
🤦